### PR TITLE
update signup modal

### PR DESCRIPTION
**Pull request by sash-binary**
- Updated modal text
- Updated email icon
- Added new page - email error info
- Updated Layout - "Main" added top padding on mobile
- Added svg icons

### CODE REVIEW

File: src/images/svg/firewalls.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...97.997 0 0 0-.986-1.007h-.592c-4.651-.208-9.18-1.822-12.982-4.64l-.378-.271a.97.97 0 0 0-1.144.008l-.355.261a23.603 23.603 0 0 1-13.027 4.642h-.55A.997.997 0 0 0 8 15.1v12.382c-.026 9.29 6.236 17.374 15.103 19.492.074.017.149.026.225.026h.345c.075 0 .15-.009.223-.026a19.326 19.326 0 0 0 7.365-3.52 1.02 1.02 0 0 0 .191-1.41.973.973 0 0 0-1.382-.195 17.373 17.373 0 0 1-6.512 3.138h-.115z"/><path fill="#FF444F" d="M52.62 50.835a7.5 7.5 0 1 1-.62-9.434v-.43c0-.536.444-.971.993-.971.548 0 .992.435.992.97v5.054a7.614 7.614 0 0 1 0 .952v1.316a3.66 3.66 0 0 0 1.15 2.654 4.077 4.077 0 0 0 2.815 1.112c1.922 0 3.567-1.386 3.973-3.19a1.004 1.004 0 0 1-.003-.243c.427-3.923-.795-7.93-3.359-11.089a14.965 14.965 0 0 0-11.787-5.535c-8.562 0-15.778 6.787-15.778 15C30.996 55.19 38.257 62 46.778 62a15.22 15.22 0 0 0 9.001-2.91.997.997 0 0 1 1.394.223 1.001 1.001 0 0 1-.222 1.396A17.212 17.212 0 0 1 46.774 64C37.184 64 29 56.325 29 47c0-9.346 8.137-17 17.764-16.999a16.954 16.954 0 0 1 13.346 6.273c2.714 3.344 4.1 7.538 3.864 11.722.02.084.029.172.025.262C63.861 51.413 61.19 54 57.95 54a6.086 6.086 0 0 1-4.203-1.665 5.714 5.714 0 0 1-1.126-1.5zM46.5 52a5.5 5.5 0 1 0 0-11 5.5 5.5 0 0 0 0 11z"/></g></svg>
```
Comment (mustofa-binary) [inline src/images/svg/firewalls.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64" aria-labelledby="firewall-icon-title"><title id='firewall-icon-title'>Firewall</title><g fill="none"><path fill="#85ACB0" d="M23.36 54.007C10.82 51.065 1.976 39.985 2.02 27.251V9.67c7.772-.36 15.226-2.948 21.48-7.445A39.923 39.923 0 0 0 44.982 9.63v17.618c0 1.197.016 2.461-.14 3.647a.999.999 0 0 0 .87 1.117 1.007 1.007 0 0 0 1.13-.859c.169-1.272.158-2.623.158-3.906V8.637c0-.55-.452-.996-1.01-.996h-.908A37.86 37.86 0 0 1 24.654.59l-.555-.4a1.02 1.02 0 0 0-1.18-.005l-.565.398a37.864 37.864 0 0 1-20.48 7.098H1.01c-.558 0-1.01.445-1.01.995v18.57C-.046 40.945 9.5 52.86 23.014 55.975c.075.017.152.026.229.026h.525a.993.993 0 0 0 .231-.027c2.658-.618 4.918-1.322 7.311-2.618a.99.99 0 0 0 .4-1.353 1.016 1.016 0 0 0-1.37-.394c-2.2 1.192-4.251 1.82-6.69 2.4h-.29z"/><path fill="#1A1A1A" d="M21 37.056v-.59c0-1.06.64-1.87 2.18-1.87s2.19.81 2.19 1.87v.59c0 1.05-.64 1.86-2.19 1.86-1.55 0-2.18-.81-2.18-1.86zm1.45-6.29l-1.16-7v-4.74a2.3 2.3 0 0 1 3.83 0v4.77l-1.13 7a.81.81 0 0 1-1.54 0v-.03z"/><path fill="#85ACB0" d="M23.443 44.987c-7.915-1.944-13.494-9.184-13.47-17.502V16.086A25.57 25.57 0 0 0 23.5 11.244a25.564 25.564 0 0 0 13.527 4.844v11.394c.002 1.8-.362 4.096-.878 5.817a1.01 1.01 0 0 0 .655 1.257.983.983 0 0 0 1.232-.668c.572-1.913.966-4.407.964-6.407V15.1a.997.997 0 0 0-.986-1.007h-.592c-4.651-.208-9.18-1.822-12.982-4.64l-.378-.271a.97.97 0 0 0-1.144.008l-.355.261a23.603 23.603 0 0 1-13.027 4.642h-.55A.997.997 0 0 0 8 15.1v12.382c-.026 9.29 6.236 17.374 15.103 19.492.074.017.149.026.225.026h.345c.075 0 .15-.009.223-.026a19.326 19.326 0 0 0 7.365-3.52 1.02 1.02 0 0 0 .191-1.41.973.973 0 0 0-1.382-.195 17.373 17.373 0 0 1-6.512 3.138h-.115z"/><path fill="#FF444F" d="M52.62 50.835a7.5 7.5 0 1 1-.62-9.434v-.43c0-.536.444-.971.993-.971.548 0 .992.435.992.97v5.054a7.614 7.614 0 0 1 0 .952v1.316a3.66 3.66 0 0 0 1.15 2.654 4.077 4.077 0 0 0 2.815 1.112c1.922 0 3.567-1.386 3.973-3.19a1.004 1.004 0 0 1-.003-.243c.427-3.923-.795-7.93-3.359-11.089a14.965 14.965 0 0 0-11.787-5.535c-8.562 0-15.778 6.787-15.778 15C30.996 55.19 38.257 62 46.778 62a15.22 15.22 0 0 0 9.001-2.91.997.997 0 0 1 1.394.223 1.001 1.001 0 0 1-.222 1.396A17.212 17.212 0 0 1 46.774 64C37.184 64 29 56.325 29 47c0-9.346 8.137-17 17.764-16.999a16.954 16.954 0 0 1 13.346 6.273c2.714 3.344 4.1 7.538 3.864 11.722.02.084.029.172.025.262C63.861 51.413 61.19 54 57.95 54a6.086 6.086 0 0 1-4.203-1.665 5.714 5.714 0 0 1-1.126-1.5zM46.5 52a5.5 5.5 0 1 0 0-11 5.5 5.5 0 0 0 0 11z"/></g></svg>
```

File: src/images/svg/spam.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...1.084-1.68l8.767 5.66c.401.131.691.512.691.961v29.13c-.056 1.754-1.504 3.132-3.21 3.083H3.238C1.504 64.043.056 62.664 0 60.878V31.781c0-.319.146-.603.374-.789z"/><path fill="#1A1A1A" d="M11 39V3.079c0-.55.42-1.003.955-1.027H52.09c.49.024.91.477.91 1.026V39h2V3.079C55.002 1.432 53.739.075 52.135 0H11.91C10.261.075 8.998 1.432 9 3.08V39h2z"/><path fill="#85ACB0" d="M22.468 44.116a1 1 0 1 1-.936 1.768l-17-9a1 1 0 1 1 .936-1.768l17 9zm36.064-9a1 1 0 0 1 .936 1.768l-17 9a1 1 0 0 1-.936-1.768l17-9z"/><path fill="#FF444F" d="M32 34c7.74 0 14-6.5 14-14.5S39.74 5 32 5s-14 6.5-14 14.5S24.26 34 32 34zm0-2c-6.619 0-12-5.588-12-12.5S25.381 7 32 7s12 5.588 12 12.5S38.619 32 32 32z"/><path fill="#FF444F" d="M32 18.143l5.324-4.88a1 1 0 1 1 1.352 1.474L33.48 19.5l5.196 4.763a1 1 0 1 1-1.352 1.474L32 20.857l-5.324 4.88a1 1 0 1 1-1.352-1.474L30.52 19.5l-5.196-4.763a1 1 0 1 1 1.352-1.474L32 18.143z"/><path fill="#85ACB0" d="M2.646 62.745c14.55-11.893 23.941-19.549 28.172-22.967.69-.52 1.675-.52 2.389.02 4.206 3.399 13.616 11.055 28.203 22.948a1 1 0 0 0 1.263-1.55c-14.588-11.895-24-19.553-28.235-22.974a4.009 4.009 0 0 0-4.851-.02c-4.26 3.442-13.654 11.1-28.206 22.994a1 1 0 0 0 1.265 1.549z"/></g></svg>
```
Comment (mustofa-binary) [inline src/images/svg/spam.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64" aria-labelledby="spam-icon-title"><title id='spam-icon-title'>Spam</title>><g fill="none"><path fill="#85ACB0" d="M.374 30.992a2.17 2.17 0 0 1 .206-.17 8.34 8.34 0 0 1 .587-.39c.207-.127.448-.272.834-.5l.512-.304.574-.34c2.295-1.37 4.225-2.602 6.333-4.102a1 1 0 1 1 1.16 1.629c-2.158 1.537-4.131 2.796-6.47 4.19l-.577.344-.512.303a52.42 52.42 0 0 0-1.021.62v28.574c.02.64.548 1.144 1.21 1.125h57.608A1.15 1.15 0 0 0 62 60.88V32.355l-8.542-5.515a1 1 0 1 1 1.084-1.68l8.767 5.66c.401.131.691.512.691.961v29.13c-.056 1.754-1.504 3.132-3.21 3.083H3.238C1.504 64.043.056 62.664 0 60.878V31.781c0-.319.146-.603.374-.789z"/><path fill="#1A1A1A" d="M11 39V3.079c0-.55.42-1.003.955-1.027H52.09c.49.024.91.477.91 1.026V39h2V3.079C55.002 1.432 53.739.075 52.135 0H11.91C10.261.075 8.998 1.432 9 3.08V39h2z"/><path fill="#85ACB0" d="M22.468 44.116a1 1 0 1 1-.936 1.768l-17-9a1 1 0 1 1 .936-1.768l17 9zm36.064-9a1 1 0 0 1 .936 1.768l-17 9a1 1 0 0 1-.936-1.768l17-9z"/><path fill="#FF444F" d="M32 34c7.74 0 14-6.5 14-14.5S39.74 5 32 5s-14 6.5-14 14.5S24.26 34 32 34zm0-2c-6.619 0-12-5.588-12-12.5S25.381 7 32 7s12 5.588 12 12.5S38.619 32 32 32z"/><path fill="#FF444F" d="M32 18.143l5.324-4.88a1 1 0 1 1 1.352 1.474L33.48 19.5l5.196 4.763a1 1 0 1 1-1.352 1.474L32 20.857l-5.324 4.88a1 1 0 1 1-1.352-1.474L30.52 19.5l-5.196-4.763a1 1 0 1 1 1.352-1.474L32 18.143z"/><path fill="#85ACB0" d="M2.646 62.745c14.55-11.893 23.941-19.549 28.172-22.967.69-.52 1.675-.52 2.389.02 4.206 3.399 13.616 11.055 28.203 22.948a1 1 0 0 0 1.263-1.55c-14.588-11.895-24-19.553-28.235-22.974a4.009 4.009 0 0 0-4.851-.02c-4.26 3.442-13.654 11.1-28.206 22.994a1 1 0 0 0 1.265 1.549z"/></g></svg>
```

File: src/images/svg/typo.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...5V30.06c0-.586-.448-1.061-1-1.061s-1 .475-1 1.06z"/><path fill="#85ACB0" d="M42.325 43.407c1.753 1.285 3.716 2.73 5.864 4.32l.303.223a3176.72 3176.72 0 0 1 14.77 10.997l.338.253a1 1 0 1 1-1.2 1.6l-.337-.252a2478.047 2478.047 0 0 0-4.362-3.261c-3.532-2.635-7.075-5.27-10.399-7.73L47 49.335c-6.555-4.847-11.387-8.367-13.71-9.967-.8-.487-1.78-.487-2.533-.03-2.37 1.63-7.202 5.15-13.757 9.997l-.302.224a3174.79 3174.79 0 0 0-14.761 10.99L1.6 60.8a1 1 0 0 1-1.2-1.6l.338-.253a2480.326 2480.326 0 0 1 4.365-3.263 3176.72 3176.72 0 0 1 10.405-7.734l.303-.224c2.147-1.588 4.11-3.033 5.864-4.319l-17.16-9.533a1 1 0 1 1 .97-1.748l17.969 9.981c2.742-1.999 4.852-3.51 6.216-4.448 1.438-.876 3.222-.876 4.707.03 1.337.92 3.434 2.424 6.17 4.418l17.967-9.981a1 1 0 0 1 .972 1.748l-17.161 9.533z"/><path fill="#FF444F" d="M32 34c7.74 0 14-6.5 14-14.5S39.74 5 32 5s-14 6.5-14 14.5S24.26 34 32 34zm0-2c-6.619 0-12-5.588-12-12.5S25.381 7 32 7s12 5.588 12 12.5S38.619 32 32 32z"/><path fill="#FF444F" d="M30 27.27v-.54c0-1 .58-1.73 2-1.73s2 .75 2 1.73v.54c0 1-.58 1.73-2 1.73s-2-.75-2-1.73zm1.2-4.82l-1.2-6.8V11a2.52 2.52 0 0 1 2-1 2.57 2.57 0 0 1 2 1v4.64l-1.18 6.8a.85.85 0 0 1-.75.54.86.86 0 0 1-.87-.53z"/></g></svg>
```
Comment (mustofa-binary) [inline src/images/svg/typo.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64" aria-labelledby="typo-icon-title"><title id='typo-icon-title'>Typo</title>><g fill="none"><path fill="#85ACB0" d="M1.7 30.71l5-4.97c.394-.393.4-1.035.014-1.436A.99.99 0 0 0 5.3 24.29l-5 4.97a1.025 1.025 0 0 0-.014 1.436.99.99 0 0 0 1.414.014zm55.6-4.97l5 4.97a.99.99 0 0 0 1.414-.014c.387-.4.38-1.043-.014-1.435l-5-4.971a.99.99 0 0 0-1.414.014c-.387.4-.38 1.043.014 1.435z"/><path fill="#1A1A1A" d="M7 33V3.131a1.19 1.19 0 0 1 1.228-1.13H55.74c.349-.01.656.106.887.323.23.218.365.518.373.834V33h2V3.131a3.19 3.19 0 0 0-3.292-3.13H8.26A3.192 3.192 0 0 0 5 3.105V33h2z"/><path fill="#85ACB0" d="M62 30.06v30.501c-.022.752-.565 1.337-1.18 1.317H3.21c-.645.02-1.188-.565-1.21-1.348V30.06C2 29.476 1.552 29 1 29s-1 .475-1 1.06v30.501c.055 1.953 1.487 3.494 3.24 3.437h57.55c1.723.057 3.155-1.484 3.21-3.405V30.06c0-.586-.448-1.061-1-1.061s-1 .475-1 1.06z"/><path fill="#85ACB0" d="M42.325 43.407c1.753 1.285 3.716 2.73 5.864 4.32l.303.223a3176.72 3176.72 0 0 1 14.77 10.997l.338.253a1 1 0 1 1-1.2 1.6l-.337-.252a2478.047 2478.047 0 0 0-4.362-3.261c-3.532-2.635-7.075-5.27-10.399-7.73L47 49.335c-6.555-4.847-11.387-8.367-13.71-9.967-.8-.487-1.78-.487-2.533-.03-2.37 1.63-7.202 5.15-13.757 9.997l-.302.224a3174.79 3174.79 0 0 0-14.761 10.99L1.6 60.8a1 1 0 0 1-1.2-1.6l.338-.253a2480.326 2480.326 0 0 1 4.365-3.263 3176.72 3176.72 0 0 1 10.405-7.734l.303-.224c2.147-1.588 4.11-3.033 5.864-4.319l-17.16-9.533a1 1 0 1 1 .97-1.748l17.969 9.981c2.742-1.999 4.852-3.51 6.216-4.448 1.438-.876 3.222-.876 4.707.03 1.337.92 3.434 2.424 6.17 4.418l17.967-9.981a1 1 0 0 1 .972 1.748l-17.161 9.533z"/><path fill="#FF444F" d="M32 34c7.74 0 14-6.5 14-14.5S39.74 5 32 5s-14 6.5-14 14.5S24.26 34 32 34zm0-2c-6.619 0-12-5.588-12-12.5S25.381 7 32 7s12 5.588 12 12.5S38.619 32 32 32z"/><path fill="#FF444F" d="M30 27.27v-.54c0-1 .58-1.73 2-1.73s2 .75 2 1.73v.54c0 1-.58 1.73-2 1.73s-2-.75-2-1.73zm1.2-4.82l-1.2-6.8V11a2.52 2.52 0 0 1 2-1 2.57 2.57 0 0 1 2 1v4.64l-1.18 6.8a.85.85 0 0 1-.75.54.86.86 0 0 1-.87-.53z"/></g></svg>
```

File: src/images/svg/work-email.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...-.617z"/><path fill="#1A1A1A" d="M5 16.688l2 .004c-.001.462.298.868.725.991 3.1.875 10.404 3.135 19.502 6l-.586 1.966c-9.073-2.857-16.373-5.115-19.451-5.983-1.297-.375-2.193-1.592-2.19-2.978zm32.402 9l-.57-1.972 19.453-5.944c.425-.13.716-.532.715-.988l2-.003c.002 1.367-.87 2.573-2.145 2.962l-19.453 5.945z"/><path fill="#FF444F" d="M28 30a2 2 0 0 1-2-2v-6a2 2 0 0 1 2-2h8a2 2 0 0 1 2 2v6a2 2 0 0 1-2 2v7h-2v-7h-4v7h-2v-7zm0-2h8v-6h-8v6z"/><path fill="#85ACB0" d="M43.305 42.44c1.5 1.152 3.138 2.414 4.901 3.777l.309.238A3128.736 3128.736 0 0 1 63.279 57.95l.338.264a1 1 0 0 1-1.234 1.574l-.337-.264a2441.482 2441.482 0 0 0-4.36-3.408c-3.53-2.753-7.072-5.507-10.395-8.077l-.308-.239c-6.555-5.067-11.386-8.745-13.721-10.423-.766-.502-1.758-.502-2.489-.024-2.37 1.702-7.201 5.38-13.756 10.447l-.308.239A3126.779 3126.779 0 0 0 1.954 59.523l-.337.264a1 1 0 1 1-1.234-1.574l.338-.264a2443.764 2443.764 0 0 1 4.363-3.41c3.532-2.756 7.076-5.511 10.401-8.084l.309-.238c1.762-1.363 3.4-2.625 4.901-3.776L4.532 33.884a1 1 0 1 1 .936-1.768l16.974 8.987c3.228-2.467 5.683-4.31 7.2-5.4 1.432-.939 3.284-.939 4.751.025 1.495 1.073 3.94 2.91 7.165 5.375l16.974-8.987a1 1 0 1 1 .936 1.768L43.305 42.44z"/></g></svg>
```
Comment (mustofa-binary) [inline src/images/svg/work-email.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64" aria-labelledby="work-email-icon-title"><title id='work-email-icon-title'>Work email</title>><g fill="none"><path fill="#85ACB0" d="M0 29.915a1 1 0 0 1 .3-.716l5-4.913a.998.998 0 0 1 1.414.015c.387.395.38 1.03-.014 1.417L2 30.336V60.72c.022.74.565 1.293 1.21 1.274h57.61c.615.019 1.158-.534 1.18-1.244V30.336l-4.7-4.618a1.004 1.004 0 0 1-.014-1.417.998.998 0 0 1 1.414-.015l5 4.913a1 1 0 0 1 .3.716V60.78c-.055 1.816-1.487 3.272-3.21 3.219H3.24c-1.753.053-3.185-1.403-3.24-3.25V29.916zM25.34 7.714a371.781 371.781 0 0 1 5.405-5.225 1.858 1.858 0 0 1 2.517.006 372.263 372.263 0 0 1 5.397 5.219 1 1 0 0 0 1.402-1.428 373.398 373.398 0 0 0-5.436-5.255 3.862 3.862 0 0 0-5.243-.006 372.93 372.93 0 0 0-5.443 5.261 1 1 0 1 0 1.402 1.428z"/><path fill="#1A1A1A" d="M7.376 8.187a1 1 0 0 1 .746-.197L8.26 8h47.48l.138-.01A1 1 0 0 1 57 8.804V33h2V8.73l-.008-.125A3 3 0 0 0 55.678 6H8.322a3 3 0 0 0-3.314 2.605L5 8.73V33h2V8.804a1 1 0 0 1 .376-.617z"/><path fill="#1A1A1A" d="M5 16.688l2 .004c-.001.462.298.868.725.991 3.1.875 10.404 3.135 19.502 6l-.586 1.966c-9.073-2.857-16.373-5.115-19.451-5.983-1.297-.375-2.193-1.592-2.19-2.978zm32.402 9l-.57-1.972 19.453-5.944c.425-.13.716-.532.715-.988l2-.003c.002 1.367-.87 2.573-2.145 2.962l-19.453 5.945z"/><path fill="#FF444F" d="M28 30a2 2 0 0 1-2-2v-6a2 2 0 0 1 2-2h8a2 2 0 0 1 2 2v6a2 2 0 0 1-2 2v7h-2v-7h-4v7h-2v-7zm0-2h8v-6h-8v6z"/><path fill="#85ACB0" d="M43.305 42.44c1.5 1.152 3.138 2.414 4.901 3.777l.309.238A3128.736 3128.736 0 0 1 63.279 57.95l.338.264a1 1 0 0 1-1.234 1.574l-.337-.264a2441.482 2441.482 0 0 0-4.36-3.408c-3.53-2.753-7.072-5.507-10.395-8.077l-.308-.239c-6.555-5.067-11.386-8.745-13.721-10.423-.766-.502-1.758-.502-2.489-.024-2.37 1.702-7.201 5.38-13.756 10.447l-.308.239A3126.779 3126.779 0 0 0 1.954 59.523l-.337.264a1 1 0 1 1-1.234-1.574l.338-.264a2443.764 2443.764 0 0 1 4.363-3.41c3.532-2.756 7.076-5.511 10.401-8.084l.309-.238c1.762-1.363 3.4-2.625 4.901-3.776L4.532 33.884a1 1 0 1 1 .936-1.768l16.974 8.987c3.228-2.467 5.683-4.31 7.2-5.4 1.432-.939 3.284-.939 4.751.025 1.495 1.073 3.94 2.91 7.165 5.375l16.974-8.987a1 1 0 1 1 .936 1.768L43.305 42.44z"/></g></svg>
```

File: src/pages/check-email/_icon-grid.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    margin: 4rem 0;
+    width: 100%;
+
+    @media ${device.tablet} {
+        grid-template-columns: none;
+    }
+`
+const GridCol = styled.article`
+    display: grid;
+    grid-template-columns: 65px 1fr;
```
Comment (mustofa-binary) [inline src/pages/check-email/_icon-grid.js]:
```suggestion
    grid-template-columns: 6.5rem 1fr;
```

File: src/pages/check-email/index.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+import SignupModal from 'components/elements/signup-modal'
+import { SectionContainer } from 'components/containers/container'
+import { localize, WithIntl } from 'components/localization'
+
+const SectionHeader = styled.p`
+    font-size: 2rem;
+    color: var(--color-black-2);
+    text-align: left;
+`
+const Container = styled.div`
```
Comment (mustofa-binary) [inline src/pages/check-email/index.js]: is this container different from the container from component?

Comment (sash-binary) [reply src/pages/check-email/index.js]: yes, it is. Actually, I added `GridContainer` to Container component.

Comment (mustofa-binary) [reply src/pages/check-email/index.js]: thats great, thank you 👍

**Pull request merged**